### PR TITLE
[RORDEV-356] filter fields not matching ror requests

### PR DESCRIPTION
**Pull request by pielas**
I don't know if such description is enough. I will think in the meantime how can I improve it. If you have some ideas please let me know

### CODE REVIEW

File: elasticsearch.md [hunk 1]
```diff
@@ -779,6 +779,8 @@ In this example, we want to avoid that users belonging to group "press" can see
 
 **⚠️IMPORTANT** The `filter`and `fields` rules will only affect "read" requests, therefore "write" requests **will not match** because otherwise it would implicitly allow clients to "write" without the filtering restriction. For reference, this behaviour is identical to x-pack and search guard.
 
+**⚠️IMPORTANT** Beginning with version 1.27.0 all ROR internal requests from kibana will not match blocks containing `filter` and/or `fields` rules.
```
Comment (coutoPL) [inline elasticsearch.md:782]: maybe we should clarify that ROR internal API is used by ROR Kibana during login & config reloading?

**Review by coutoPL — CHANGES_REQUESTED** [review]

File: elasticsearch.md (hunk 1 above)
Comment (sscarduzio) [reply elasticsearch.md:782]: I think it's worth to show the example from tests you showed us in the Jira ticket. It very much gives the idea of the ramifications this change has.

Comment (pielas) [reply elasticsearch.md]: I added some additional info and changed examples of "block duplication strategy"

**Review by coutoPL — APPROVED** [review]
For me it's OK. @sscarduzio please let us know if it's also ok for you or maybe there is sth you'd like to add?

**Comment by sscarduzio** [comment]
LGTM, thanks

**Pull request merged**